src/pyodide/internal/builtin_wrappers.js

```diff
@@ -0,0 +1,88 @@
+import { default as Eval } from "pyodide-internal:eval";
```

Comment:
Note that we already have the `UnsafeEval` object (see `api/unsafe.h`). It is currently used only in a binding but could also be exposed by the `pyodide-internal:eval` import here.

Reply:
I'm not sure what you mean. As a guess are you suggesting adding `UnsafeEval::newWebAssemblyModule`? That seems reasonable.

Reply:
Okay I did this.